docs/reference/cli/ocm_completion_fish.md

```diff
@@ -28,6 +28,39 @@ ocm completion fish [flags]
       --no-descriptions   disable completion descriptions
 ```
 
+### Options inherited from parent commands
+
+```
+      --config string    supply configuration by a given configuration file.
+                         By default (without specifying custom locations with this flag), the file will be read from one of the well known locations:
+                         1. The path specified in the OCM_CONFIG_PATH environment variable
+                         2. The XDG_CONFIG_HOME directory (if set), or the default XDG home ($HOME/.config), or the user's home directory
+                         - $XDG_CONFIG_HOME/ocm/config
+                         - $XDG_CONFIG_HOME/.ocmconfig
+                         - $HOME/.config/ocm/config
+                         - $HOME/.config/.ocmconfig
+                         - $HOME/.ocm/config
+                         - $HOME/.ocmconfig
+                         3. The current working directory:
+                         - $PWD/ocm/config
+                         - $PWD/.ocmconfig
+                         4. The directory of the current executable:
+                         - $EXE_DIR/ocm/config
+                         - $EXE_DIR/.ocmconfig
+                         Using the option, this configuration file be used instead of the lookup above.
+      --logformat enum   set the log output format that is used to print individual logs
+                            json: Output logs in JSON format, suitable for machine processing
+                            text: Output logs in human-readable text format, suitable for console output (must be one of [json text]) (default json)
```

Comment:
This looks a bit odd. Shouldn't this `(must be one of [json text]) (default json)` be on L51?

Reply:
The same is true for `loglevel` and `logoutput`

Reply:
Why should this be on this line? The enum specification refers to https://github.com/open-component-model/open-component-model/blob/867a2779a47c1bbb020d0bdbd8e9217536ce2ccb/cli/internal/enum/flag.go#L56 so the `must be one of...` string comes always after the flag description. The `default ...` comment is added by cobra for the default value of the flag.

That said, because the flag description doesn't stop in the first line I wouldnt expect it on L51

Reply:
I adopted the second one because the first one is not possible without changing the way that enum argument parsing works and I would like to not complicate things